inventory/host_vars/staging.coopcircuits.fr/config.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+domain: staging.coopcircuits.fr
+host_id: fr-staging
+rails_env: staging
+
+admin_email: admin@coopcircuits.fr
+
+mail_domain: coopcircuits.fr
+
+unicorn_timeout: 120
+
+users_sysadmin:
+  - "{{ core_devs }}"
+  - paco
+  - hugo
```

Comment:
I think we can remove hugo for now, we can re-add him if he comes back at some point in the future :+1: